ansible/services.yml

```diff
@@ -76,11 +76,16 @@
          - listen 80 default_server
          - listen 443 ssl http2
          - server_name _
-         - root /app/web
+         - |
+          set $root /app/web;
```

Comment:
What if we have this as extra_var? Default should be the new path?

Reply:
this var is nginx. do you mean to try to add it as ansible var?

Reply:
Yeah, ansible var used in this config